python/AppTop/AppTop.py

```diff
@@ -20,14 +20,13 @@
 import time
 import pyrogue   as pr
 import AppTop    as appTop
+import AppCore
```

Comment:
From  python/AppTop/__init__.py, `from AppTop.AppCore  import *`, so I don't think you need this import because you already have `import AppTop    as appTop`

Reply:
Yes that is valid.

Reply:
I modified that line.